cmd/config/subcommand/sandbox/sandbox_config.go

```diff
@@ -2,7 +2,9 @@ package sandbox
 
 //go:generate pflags Config --default-var DefaultConfig --bind-default-var
 var (
-	DefaultConfig = &Config{}
+	DefaultConfig = &Config{
+		Version: "dind",
```

Comment:
The example shows version number 0.17.0 whereas here the default is dind. 
Thats confusing .

Reply:
Also mention how the users picks these versions from

Reply:
default is latest because we want to use `dind` image. It's equivalent to `latest`

Reply:
Users can pass any version that is available in GitHub release

Reply:
Lets mention the link to release page here . and can we not use latest .

Reply:
latest tag doesn't have dind image

Reply:
because of this https://github.com/flyteorg/flytectl/pull/177#discussion_r704163922 i reverted the default value and added a comment